.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
```

Comment:
Should this be `Test`?

It's showing with the same name as the old `Build` pipeline in _Actions_.